### PR TITLE
Fixes #13363 - Adding additional qpid debug statements to foreman-debug

### PR DESCRIPTION
**Pull request by mccun934**

### CODE REVIEW

File: katello/katello-debug.sh [hunk 1]
```diff
@@ -13,6 +13,22 @@ then
...
+  for FILE in $*; do
+    echo "Copying entire file: $FILE"
+    if [ \( -f "$FILE" -o -h "$FILE" \) -a \( -r "$FILE" -a -s "$FILE" \) ]; then
+      printv " - $FILE"
+      SUBDIR=$(dirname $FILE)
+      [ ! -d "$DIR$SUBDIR" ] && mkdir -p "$DIR$SUBDIR"
+      cp "$FILE" "$DIR$FILE"
+    fi
+  done
+}
```
Comment (jlsherrill) [inline katello/katello-debug.sh:28]: curious if we've discussed possibly changing this behavior as it can be really annoying for files it captures

Comment (ehelms) [inline katello/katello-debug.sh:28]: @mccun934 ping

Comment (mccun934) [inline katello/katello-debug.sh:28]: nope, never discussed it beyond here. For some files it is required to copy the whole thing, namely the tasks-export, truncating ZIP/tar isn't really helpful.

**Comment by ehelms** [comment]
@jlsherrill please re-review

**Comment by jlsherrill** [comment]
ACK thanks @mccun934 !

**Pull request merged**